students/19th/saemijung/JS/login.js

```diff
@@ -0,0 +1,18 @@
+
+const thisIsPw = document.getElementById('login_pw');
+const thisIsId = document.getElementById('login_id');
+
+thisIsPw.addEventListener("keydown",function(){
+    let inputId = thisIsId.value;
+    let inputpw = thisIsPw.value;
+    
+    if(inputId.length>=1 && inputpw.length>=1)
```

Comment:
이부분에서 변수명 inputId가 저라면 dom객체라고 착각할 수도 있을 것 같은데 
```suggestion
const inputpw = document.getElementById('login_pw');
const inputId = document.getElementById('login_id');

thisIsPw.addEventListener("keydown",function(){
    
    if(inputId.value.length>=1 && inputpw.value.length>=1)
```
로 바꾸면 어떨까요?

Reply:
종호센세님~수정했습니다! 감사해요~:D